docs/api/base-window.md

```diff
@@ -0,0 +1,1403 @@
+# BaseWindow
+
+> Create and control windows.
+
+Process: [Main](../glossary.md#main-process)
+
+> **Note**
+> `BaseWindow` provides a flexible way to compose multiple web views in a
+> single window. For windows with only a single, full-size web view, the
+> [`BrowserWindow`](browser-window.md) class may be a simpler option.
+
+This module cannot be used until the `ready` event of the `app`
+module is emitted.
+
+```javascript
+// In the main process.
+const { BaseWindow, WebContentsView } = require('electron')
+
+const win = new BaseWindow({ width: 800, height: 600 })
+
+const leftView = new WebContentsView()
+leftView.webContents.loadURL('https://electronjs.org')
+win.contentView.addChildView(leftView)
+
+const rightView = new WebContentsView()
+rightView.webContents.loadURL('https://github.com/electron/electron')
+win.contentView.addChildView(rightView)
+
+leftView.setBounds({ x: 0, y: 0, width: 400, height: 600 })
+rightView.setBounds({ x: 400, y: 0, width: 400, height: 600 })
+```
+
+## Parent and child windows
+
+By using `parent` option, you can create child windows:
+
+```javascript
+const { BaseWindow } = require('electron')
+
+const parent = new BaseWindow()
+const child = new BaseWindow({ parent })
+```
+
+The `child` window will always show on top of the `parent` window.
+
+## Modal windows
+
+A modal window is a child window that disables parent window, to create a modal
+window, you have to set both `parent` and `modal` options:
```

Comment:
```suggestion
A modal window is a child window that disables its parent window. To create a modal
window, you have to set both the `parent` and `modal` options:
```

Reply:
This is copy/pasted from the BrowserWindow section by the same name. I'll fix this in both places now, but we might want to consider how to deduplicate this sort of info.